src/sidebar/components/annotation-publish-control.js

```diff
@@ -34,7 +49,12 @@ function AnnotationPublishControl({
 
   const isPrivate = draft ? draft.isPrivate : !isShared(annotation.permissions);
 
-  const publishDestination = isPrivate ? 'Only Me' : group.name;
```

Comment:
Since you can't post an annotation if there is no focused group, I think it would be easier just to render nothing in this case, by adding the following after the last hook call:

```js
  if (!group) {
    return null;
  }
```

The lazier option would be to just assume that the group is successfully fetched and cast the result of the `useStore(store => store.getGroup(...))` call to `Group`. This does create the risk of a runtime error if for any reason this fails though!

Reply:
Looking at https://github.com/hypothesis/client/issues/1542 there was an issue in the past, that may still be there, where an annotation card did somehow end up being rendered when there was no focused group. If that issue still exists, then clicking the "Edit" button in that state would cause an exception here.